src/devices/Units/Pressure.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Units
+{
+    /// <summary>
+    /// Structure representing pressure
+    /// </summary>
+    public struct Pressure
+    {
+        private const double MbarRatio = 0.01;
+        private const double KpaRatio = 0.001;
+        private const double HpaRatio = 0.01;
+        private const double InhgRatio = 0.000295301;
+        private const double MmhgRatio = 0.00750062;
+        private double _pa;
+        
+        private Pressure(double pa)
+        {
+            _pa = pa;
+        }
+        
+        /// <summary>
+        /// The mean sea-level pressure (MSLP) is the average atmospheric pressure at mean sea level.
+        /// </summary>
+        public static Pressure MeanSeaLevel => Pressure.FromPa(101325);
+        
+        /// <summary>
+        /// Pressure in Pa
+        /// </summary>
+        public double Pa => _pa;
+        
+        /// <summary>
+        /// Pressure in mbar
+        /// </summary>
+        public double Mbar => MbarRatio * _pa;
```

Comment:
I think the casing should be `MBar`

Reply:
Actually there's a problem. The metric unit for millibar is mbar (everything lowercase); that's why I used Mbar. However Mbar (with the capital M) stands for megabar. Therefore I think I should use Millibar and MillibarRatio, do you agree?